src/test/resources/com/puppycrawl/tools/checkstyle/checks/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -102,4 +102,17 @@ void foo() {
 	private String notAUnicodeEscaped2 = "\\\\u1234";
 
 	private String onlyEscaped = "\\\u1234";
+
+	private String controlC0 = ""
+			+ "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\n\u000b\u000c\r\u000e\u000f"
```

Comment:
The missing unicode above are `\u000a` (`\n`) and `\u000d` (`\r`). In Java, unicode escapes are pre-processed before the compiler is run. Therefore, if you put `\u000a` in a String literal like 
```
String someString = "foo\u000Abar";
```
It will be compiled exactly as if you wrote
```
String someString = "foo
bar";
```
This can not even be compiled successfully. Unlike other common control characters (i.e. a Greek letter mu), these two may lead to more problems like compilation error. [Google Style](http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s2.3.3-non-ascii-characters) said `The choice depends only on which makes the code easier to read and understand`. IMO, we shouldn't allow user to use these two characters. Google Style allows using escaped characters of non-printable ones, but not misleading ones. And in this case the escaped format of line feed and carriage return can be `\n` and `\r`, instead of `\u000a` and `\u000d`.